zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
```

Comment:
Thinking about it more I think we should replace `checkbox` with `allow_multiple`, also add an `allow_none` flag.
`allow_none` means that you offer a none option when picking from the list, or allow no checkboxes to be selected. This should simplify your logic elsewhere.